flake.nix

```diff
@@ -327,6 +327,10 @@
                   if [ "$(ulimit -Sn)" -lt "1024" ]; then
                       >&2 echo "⚠️  ulimit too small. Run 'ulimit -Sn 1024' to avoid problems running tests"
                   fi
+
+                  if [ -z "$(git config --global merge.ours.driver)" ]; then
+                      >&2 echo "⚠️  Recommended to run 'git config --global merge.ours.driver true' to enable better lock file handling. See https://blog.aspect.dev/easier-merges-on-lockfiles for more info"
+                  fi
```

Comment:
can we set this in repo local config

Reply:
Possibly, but changes like that don't propagate with `git clone` (because `true` here is an arbitrary command) so I don't think it changes much. And if we/you/someone e.g. want to use it in multiple projects, it's easier to do it once in global user's config than everywhere.